core/ecsql/common/eslint.config.js

```diff
@@ -0,0 +1,25 @@
+const iTwinPlugin = require("@itwin/eslint-plugin");
+const eslintBaseConfig = require("../../../common/config/eslint/eslint.config.base");
+
+module.exports = [
+  {
+    files: ["**/*.ts"],
+    ...iTwinPlugin.configs.iTwinjsRecommendedConfig,
+  },
+  {
+    files: ["**/*.ts"],
```

Comment:
can delete this extension config

Reply:
dont forget this;
just delete the second object from this array we are exporting